lienzo-webapp/src/main/java/org/kie/lienzo/client/BaseExample.java

```diff
@@ -68,8 +70,16 @@ public void init(final LienzoPanel panel, HTMLDivElement topDiv) {
 
         MousePanMediator pan = new MousePanMediator(EventFilter.META);
         this.panel.getViewport().pushMediator(pan);
+
+        jsLienzo = new JsLienzo(this.panel, this.layer);
+        setupJsLienzoTypes(jsLienzo);
     }
 
+    // TODO: Move to J2CL impl
+    public static native void setupJsLienzoTypes(Object jsLienzo) /*-{
```

Comment:
jsni to j2cl, pleaes complete the TODO above

Reply:
@romartin done

Reply:
thx